autoload/utils/exec/system.vim

```diff
@@ -9,6 +9,13 @@ function! utils#exec#system#run(cmd, open_qf, errFormat) abort
     endif
     let l:s_out = system(a:cmd)
     let l:ret_code = v:shell_error
+
+    if l:ret_code == 0
+        echon 'Success!'
+    else
+        echon 'Failure!'
```

Comment:
Here you are using `echon`

Reply:
Should I change it to `echom` or `echon`?

I tried using it with `echon` and the message showed only for a split second, but with `echom` it stays shown and it is in `:messages`.

Reply:
In general in the plugin I always use `echon` to print notification messages (for example that CMakeCache was removed or something like that). I propose to use one approach in the plugin. If you have any comments why do we need to use always `echom` let's review and discuss it.

Reply:
I have no particular reason for `echom`, I pushed changes to `echon` now.